lib/pages/bucket_page/bloc/bucket_page_bloc.dart

```diff
@@ -22,10 +22,8 @@ class BucketPageBloc extends Bloc<BucketPageEvent, BucketPageState> {
     ToBack event,
     Emitter<BucketPageState> emit,
   ) {
-    List<String> splittedPath = state.path.split('/');
-    String lastDirectory = splittedPath.elementAt(splittedPath.length - 2);
-
-    String newPath = state.path.replaceAll('$lastDirectory/', '');
+    List<String> newPath = [...state.path];
+    newPath.removeLast();
```

Comment:
maybe we have method for copy without last elem?

Reply:
> maybe we have method for copy without last elem?

` List<String> newPath = state.path.sublist(0, state.path.length - 1);`

That's better?